src/main/java/com/google/devtools/build/lib/skyframe/ArtifactFunction.java

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Bazel Authors. All rights reserved.
+// Copyright 2019 The Bazel Authors. All rights reserved.
```

Comment:
We should not be updating copyright notices. The date of first release remains corect.